test/image-blueprints/group3/rhel92-source-offline.toml

```diff
@@ -0,0 +1,46 @@
+name = "rhel-9.2-microshift-source-offline"
```

Comment:
What's different about this blueprint and the existing isolated one?

Reply:
Nothing actually, except for the name. 
I was not sure if I can use a blueprint from another group. Let me try this asap.

Reply:
It seems to work only if `group2` cache exists. Since we have an option to rebuild a specific group, I moved the isolated toml to `group3`. This should also allow us to optimize the builds because `group3` will only be used in periodics, so we can save on nightly builds.
WDYT?